### DISCUSSION TITLE
Add button to skip "We think you’re gonna like it here"

### DISCUSSION THREAD

**Discussion by landsman** [community/community#61458] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hello GitHub team,

Could you please add an option to skip this entire wizard? It's making me frustrated having to click through every step without the ability to skip it. I truly don't need to go through these learning steps, so I kindly request that you provide an option for non-beginner users. 🙏

Thank you!

---

Steps to reproduce it:

1. create a new organization
2. on overview tab is this wizard with `We think you’re gonna like it here.`...
3. try to skip it...

**Comment by landsman** [comment]
Oh, it's there - on the right side, totaly invisible! 😢 
Please put it to the left side, under or after this wizard 🙏

**Reply by tristanpoland** [reply]
I second this, thanks for mentioning that there was on because I looked for a good 5 mins. 100% needs to be changed to a button at the bottom of the wizard.

**Comment by ScottRFrost** [comment]
For people who ended up here like me and were STILL confused:

![image](https://github.com/community/community/assets/3866418/ad9bc8b0-8ded-49d2-962a-b565145b470a)
